ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -150,16 +150,20 @@ def _extract_output_message(self, response):
 
         elif isinstance(content, list):
             for completion in content:
+                tool_calls_info = []
                 text = _get_attr(completion, "text", None)
                 if isinstance(text, str):
                     output_messages.append({"content": text, "role": role})
                 else:
                     if _get_attr(completion, "type", None) == "tool_use":
-                        name = _get_attr(completion, "name", "")
-                        inputs = _get_attr(completion, "input", "")
-                        output_messages.append(
-                            {"content": "[tool: {}]\n\n{}".format(name, json.dumps(inputs)), "role": role}
-                        )
+                        tool_call_info = {
+                            "name": _get_attr(completion, "name", ""),
+                            "arguments": _get_attr(completion, "input", ""),
+                            "tool_id": _get_attr(completion, "id", ""),
+                            "type": _get_attr(completion, "type", ""),
+                        }
+                        tool_calls_info.append(tool_call_info)
```

Comment:
Why are we appending tool call info to a list if we're just directly adding the tool call info to the output messages?

Reply:
Following up on offline discussion - anthropic currently supports one tool use dictionary per content block.